src/neptune_scale/sync/files.py

```diff
@@ -29,6 +39,10 @@ def guess_mime_type_from_file(local_path: Union[pathlib.Path, str], destination:
     that the file is inaccessible or not found, thus the upload will fail.
     """
     try:
+        for extra_ext, extra_mime in EXTRA_TYPES.items():
```

Comment:
Why make it lazy? IMO should be eager on loading the module. 
Maybe not ideal - we're polluting a global state - but IMO easily good enough for now.